<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>thirdwindow</class>
 <widget class="QWidget" name="thirdwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableInventaris">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="rowCount">
    <number>8</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Nama</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Kategori</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>-50</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>User</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>Urutkan :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbSort">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Nama</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Kategori</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jumlah</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnUrut">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Urutkan</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
